.travis.yml

```diff
@@ -39,10 +39,6 @@ matrix:
       env: TOXENV=py37
       dist: xenial
       sudo: required
-    - python: 3.8-dev
-      env: TOXENV=py38
-      dist: xenial
-      sudo: required
```

Comment:
If you remove this section, the 3.8-dev tests won't run at all. We want to run them, and at the same time allow them to fail.